### ISSUE TITLE
Personal Search History has been replaced with Trending Searches

### ISSUE THREAD

**Issue by cortan4** [yuliskov/SmartTubeNext#2916]
Whenever I use the Search Function, all of my _personal_ most recent search history would be persistent and available.  Now, my personal search history is continually being replaced with a list of Trending Searches.  Now, every time I need to re-enter a search from scratch, rather than selecting from my search history, which is no longer there.

**Comment by Skyber1967** [comment]
Same here

**Comment by Abnormalia** [comment]
Same here started 1 week ago, very annoying

**Comment by DiscoFever78** [comment]
Yeah, been retyping my common searches about a dozen times a day this past week.

**Comment by Chebulex** [comment]
Same here. Updated it today to 19.52 which according to the changelog "fixes search history issues".
The trending searches are gone now, but the search history only displays searches that have been made on the device. Searches that were done on the PC or Smartphone are no longer displayed in SmartTube which is pretty bad.

**Comment by kapilmahawar** [comment]
> Same here. Updated it today to 19.52 which according to the changelog "fixes search history issues". The trending searches are gone now, but the search history only displays searches that have been made on the device. Searches that were done on the PC or Smartphone are no longer displayed in SmartTube which is pretty bad.

atlease it's far better than typing everything every single time. most of my TV searches are TV specific, so no issues for me. but i can understand someelse getting frustrated with such arrangement.

**Comment by Blackhawke** [comment]
I just found and switched from Youtube to SmartTube yesterday. So I can report this was a YouTube thing, not a SmartTube thing, as it happened there too and was very annoying.

It went away a few days ago, however, and we had our search history back. Unfortunately, when we did the switch it didn't follow to SmartTube. So guess searches are device and app specific.
